packages/framework-integration-tests/src/read-models/cart-read-model.ts

```diff
@@ -21,6 +22,16 @@ export class CartReadModel {
   public getChecks() {
     return this.checks
   }
+
+  public static beforeFn(filter: FilterFor<CartReadModel>, currentUser?: UserEnvelope): FilterFor<CartReadModel> {
+    return filter
+  }
+
+  public static beforeFnV2(filter: FilterFor<CartReadModel>, currentUser?: UserEnvelope): FilterFor<CartReadModel> {
+    if (!filter.id || filter.id?.eq !== 'before-fn-test') return filter
+    return { id: { eq: filter.id.eq + '-modified' } } as FilterFor<CartReadModel>
+  }
+
```

Comment:
This is an amazing way to test before filters. Just one comment: Don't we need a test case where an exception is thrown inside a before filter, so that the read model query doesn't return results?

Reply:
Yes! I forgot about that, thanks!